analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Sonar/SonarExplodedGraph.cs

```diff
@@ -715,8 +715,6 @@ private ProgramState EnsureStackState(ExpressionSyntax parenthesizedExpression,
             if (ShouldConsumeValue(parenthesizedExpression))
             {
                 var newProgramState = programState.PopValue();
-                System.Diagnostics.Debug.Assert(!newProgramState.HasValue);
-
```

Comment:
This would fail on local IT run and annoy us more than it needs to.

We will never try to fix the design issue with CFG and C# 8+.